### PR TITLE
ES Client logging middleware

### PR DESCRIPTION
**Pull request by miiu96**
Added in the `elasticsearch` client a custom logger that will print all the HTTP requests and responses.

### CODE REVIEW

File: client/logging/customLogger.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+)
+
+var log = logger.GetOrCreate("indexer/client/requests")
+
+type CustomLogger struct{}
```
Comment (bogdan-rosianu) [inline client/logging/customLogger.go:14]: add comments for exported stuff

File: client/logging/customLogger.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	_ time.Time,
+	dur time.Duration,
+) error {
+	var (
+		nReq int64
+		nRes int64
+	)
+
+	if req != nil && req.Body != nil && req.Body != http.NoBody {
+		nReq, _ = io.Copy(ioutil.Discard, req.Body)
```
Comment (bogdan-rosianu) [inline client/logging/customLogger.go:29]: rename `nReq` and `nRes` to something else

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]
small stuff. good PR ! :medal_sports:

File: client/logging/customLogger.go (hunk 1 above)
Comment (miiu96) [reply client/logging/customLogger.go:14]: done.

File: client/logging/customLogger.go (hunk 2 above)
Comment (miiu96) [reply client/logging/customLogger.go:29]: done.

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**